### ISSUE TITLE
Error while compiling

### ISSUE THREAD

**Issue by starcoat** [davidgiven/wordgrinder#32]
(FC25, references for "lua5.2" changed to "lua" in Makefile)

make -j fails with this error:

```
Lua error: [string "src/lua/fileio.lua"]:514: malformed property .documents.1.co: 6.0
stack traceback:
	[C]: in function 'error'
	[string "src/lua/fileio.lua"]:514: in function 'loadfromstreamt'
	[string "src/lua/fileio.lua"]:574: in function 'LoadFromStream'
	[string "src/lua/fileio.lua"]:581: in upvalue 'loaddocument'
	[string "src/lua/fileio.lua"]:607: in field 'LoadDocumentSet'
	tests/weirdness-save-new-document.lua:15: in local 'f'
	[string "src/lua/main.lua"]:260: in local 'fn'
	[string "src/lua/main.lua"]:324: in function 'Main'
```

**Comment by davidgiven** [comment]
I think you're using Lua 5.3 --- it now distinguishes between ints and reals, so the number 6 can turn into "6.0" or "6" depending on the representation.

Could you do `lua -v` and tell me what you get, please?

**Comment by starcoat** [comment]
I'm on Lua 5.3.4

**Comment by davidgiven** [comment]
Good, thought so.

 WordGrinder only supports Lua 5.1 and 5.2 and LuaJIT. It does not support 5.3 --- that's why the Makefile explicitly refers to 5.2.

I'll add support for 5.3, but for now if you want it to work, you'll need to install 5.2 and then tell the Makefile where the 5.2 libraries are.

**Comment by davidgiven** [comment]
...so porting to use 5.3 wasn't hard; it's in head of git. If you want to use a released version, you'll still need 5.2, but if you get head of git you can edit the makefile the way you've already done and it should, hopefully, build.

Thanks for the bug report!

**Comment by starcoat** [comment]
Works like a charm!
Thank you for fixing this so fast :)
